src/components/TargetForm/TargetFormContent.js

```diff
@@ -137,11 +138,13 @@ const TargetFormContent = ({
       {existent ? (
         <View style={styles.buttonsContainer}>
           <Button
+            handleOnPress={onDelete}
             additionalStyles={[styles.editButton, { backgroundColor: RED }]}
             title={deleteButton}
             isLoading={targetStatus === LOADING}
           />
           <Button
+            handleOnPress={() => null}
```

Comment:
This will be the edit target feature. It is not implemented yet.

Reply:
A good idea is to leave some `TODO` comments, so the reviewers can see and you can search for TODO tags later